java/src/test/java/ai/rapids/cudf/BinaryOpTest.java

```diff
@@ -35,6 +35,7 @@
   private static final Double[] DOUBLES_2 = new Double[]{10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 100.0};
   private static final Boolean[] BOOLEANS_1 = new Boolean[]{true, true, false, false, null};
   private static final Boolean[] BOOLEANS_2 = new Boolean[]{true, false, true, false, true};
+  private static final int[] SHIFT_BY = new int[]{1, 2, 3, 4, 5, 10, 20};
```

Comment:
Ideally these shift values for testing should include some unusual cases like 0, small negative values, and large values both on the positive and negative side.

Reply:
You are right, these tests are not exhaustive, but mostly to test the plumbing. I can make the change if you still want